flink-table/flink-table-storage/src/main/java/org/apache/flink/table/storage/filestore/lsm/sst/SstFile.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.storage.filestore.lsm.sst;
+
+import org.apache.flink.api.common.serialization.BulkWriter;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.connector.file.src.FileSourceSplit;
+import org.apache.flink.connector.file.src.reader.BulkFormat;
+import org.apache.flink.connector.file.src.util.RecordAndPosition;
+import org.apache.flink.core.fs.FSDataOutputStream;
+import org.apache.flink.core.fs.FileSystem;
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.data.binary.BinaryRowData;
+import org.apache.flink.table.runtime.typeutils.RowDataSerializer;
+import org.apache.flink.table.storage.filestore.KeyValue;
+import org.apache.flink.table.storage.filestore.KeyValueSerializer;
+import org.apache.flink.table.storage.filestore.stats.FieldStatsCollector;
+import org.apache.flink.table.storage.filestore.utils.FileFormat;
+import org.apache.flink.table.storage.filestore.utils.FileUtils;
+import org.apache.flink.table.storage.filestore.utils.RecordIterator;
+import org.apache.flink.table.storage.filestore.utils.RecordReader;
+import org.apache.flink.table.types.logical.RowType;
+
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * This file includes several {@link KeyValue}s, representing a portion of changes applied to the
+ * table.
+ */
+public class SstFile {
+
+    private final Configuration conf;
+    private final RowType keyType;
+    private final RowType valueType;
+
+    private KeyValueSerializer serializer;
```

Comment:
We can keep a final `SstFile`. Put these vars to the reader.
And we should consider this situation:
```
var reader1 = SstFile.read(...);
var reader2 = SstFile.read(...);
merge(reader1, reader2)
```
This should be a problem? Add the test.